src/main/java/forestry/farming/logic/crops/CropRubber.java

```diff
@@ -43,6 +45,9 @@ public CropRubber(World world, IBlockState blockState, BlockPos position) {
 	private static <T extends Comparable<T>> IBlockState getReplantState(IBlockState sappyState) {
 		if (hasRubberToHarvest(sappyState)) {
 			for (Map.Entry<IProperty<?>, Comparable<?>> wetPropertyEntry : sappyState.getProperties().entrySet()) {
+				if (wetPropertyEntry.getKey() instanceof PropertyBool && wetPropertyEntry.getKey().getName().equals("hassap")) {
```

Comment:
Below this we have `IProperty<?> property = wetPropertyEntry.getKey();`, you should use `property` here instead of calling `getKey()` more times.